providers/Tidal/v2/lookup.ts

```diff
@@ -125,6 +125,13 @@ export class TidalV2ReleaseLookup extends ReleaseApiLookup<TidalProvider, Single
 			next = content.links.next;
 		}
 
+		if (items.length < rawRelease.data.attributes.numberOfItems) {
+			this.addMessage(
+				`The API returned only ${items.length} of ${rawRelease.data.attributes.numberOfItems} tracks.`,
```

Comment:
```suggestion
		const realTrackCount = rawRelease.data.attributes.numberOfItems;
		if (items.length < realTrackCount) {
			this.addMessage(
				`The API returned only ${items.length} of ${realTrackCount} tracks for ${this.lookup.region}, other regions may have more`,
```
We can also mention the looked up region and give a hint that trying other regions sometimes helps.
I've dropped the full stop as other messages don't have one either, some are not even full sentences.